deephaven/web-client-ui
# fix: Handle deletion of unsaved copied file in NotebookPanel

* Previously, duplicating a notebook file and then trying to delete it without saving it first will cause an error to occur
* Added a `fileExists` function to `FileUtils` to handle this check
* Added a `getUniqueCopyFileName` function to `FileUtils`
* Behaviour of copying a file from `NotebookPanel` changed from opening an unsaved copy of the file to duplicating the file in `FileStorage` first
* Related to #1359 

#### Testing Instructions:
* Open a file in the notebook panel
* Copy the opened file by using the 'Copy File' option in the overflow menu
* Delete the file by using the overflow menu (No error should occur)
* Repeat the same steps but copy the file by right clicking the file in the file explorer
* Delete the copied file again, this time it should also disappear from the file explorer
